pkg/kubectl/cmd/util/factory_object_mapping.go

```diff
@@ -369,8 +381,8 @@ func (f *ring1Factory) AttachablePodForObject(object runtime.Object) (*api.Pod,
 	return pod, err
 }
 
-func (f *ring1Factory) PrinterForMapping(cmd *cobra.Command, mapping *meta.RESTMapping, withNamespace bool) (kubectl.ResourcePrinter, error) {
-	printer, generic, err := PrinterForCommand(cmd)
+func (f *ring1Factory) PrinterForMapping(cmd *cobra.Command, mapping *meta.RESTMapping, withNamespace bool) (printers.ResourcePrinter, error) {
+	printer, generic, err := f.PrinterForCommand(cmd)
```

Comment:
No peer references.  You'll need to push this out further.  Also, composition models suck compared to inheritance.